_config.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+use Psr\Log\LoggerInterface;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\NonBlockingSessions\FileSessionHandler;
+
+// PHPUnit 9 outputs headers early - but since we're catching this scenario we should log it just in case.
```

Comment:
Where's this scenario being caught?

Please update inline comment to explain

Reply:
We're catching it right here. What do you mean?

Reply:
It's still very confusing

Why is this even here? i.e doesn't it already get logged by default?

Why do we have "Logic to handle unit-test in code"? (which we were just trying to avoid [here](https://github.com/silverstripe/silverstripe-elemental/pull/1364/files#r2217993011) in an unrelated PR)?

Reply:
It's here because 
> PHPUnit 9 outputs headers early

That causes an error if we try to call `session_set_save_handler()` after headers have been sent.

> Why do we have "Logic to handle unit-test in code"? 

Because it's unfortunately unavoidable in PHPUnit 9 because of the way PHPUnit itself is implemented. I did try to handle this in the test but couldn't find a way to do so. I then discussed this with you offline.

Reply:
I've updated the comment again. Let me know if you still want more detail, and if so, what detail should be added.